pkg/query/aggapi.go

```diff
@@ -20,8 +20,9 @@ type aggCostModelResponse struct {
 }
 
 // QueryAggCostModel queries /model/aggregatedCostModel by proxying a request to Kubecost
-// through the Kubernetes API server.
-func QueryAggCostModel(clientset *kubernetes.Clientset, kubecostNamespace, serviceName, window, aggregate, aggregationSubfield string, ctx context.Context) (map[string]Aggregation, error) {
+// through the Kubernetes API server if useProxy is true or, if it isn't, by
+// temporarily port forwarding to a Kubecost pod.
+func QueryAggCostModel(restConfig *rest.Config, kubecostNamespace, serviceName, window, aggregate, aggregationSubfield string, useProxy bool, ctx context.Context) (map[string]Aggregation, error) {
```

Comment:
With these invocations getting more complicated, what do you think about passing a struct for named arguments?

Reply:
Thank you for the fix - looking forward to GCP working soon. I agree that that's a lot of arguments and could cause some confusion to check argument placement etc. So a struct would seem like a good call.

Reply:
Second (third?) the idea of a struct for named arguments.